### ISSUE TITLE
Very good coding approach

### ISSUE THREAD

**Issue by informatic-revolution** [yceruto/symfony-ddd-skeleton#16]

**Comment by informatic-revolution** [comment]
Thank you for sharing this very interesting coding approach. I would like to try it using a private and public api sharing the same core.

Is there a way to use symfony server and run the two kernels at the same time ?

**Comment by yceruto** [comment]
Hi, Thanks!

> Is there a way to use symfony server and run the two kernels at the same time ?

I didn't tried yet, but if the symfony server allow you to run more than one instance (with different ports or domains) and set the `APP_NAME` env var, you should be able. At least it works using the builtin PHP server.

However, I need to check again with the latest changes in the front, because the `.env` file is being prioritized and it's probable that prefixing the env var is not working right now.

**Comment by informatic-revolution** [comment]
Thank you for the answer, this is what I get when I run the second kernel

 [WARNING] The local web server is already running                                                                      
                                                                                                                        
Local Web Server
    Listening on http://127.0.0.1:8004
    The Web server is using PHP FPM 7.4.11

Basically, it looks like Symfony uses the project name to know if a server is already running. 

I was able to launch a second server creating a symlink, but this is a hack and I was wondering if there is another way to accomplish it.

Thank you in advance for your help

**Comment by LucileDT** [comment]
I found a way to implement this using the Symfony server, are you interested in some documentation detailing how to do so?

**Comment by yceruto** [comment]
Hello @LucileDT can you update the README.md file at the root of the repo to document that? thanks!

**Comment by yceruto** [comment]
I've released a new version containing the doc that solves this ticket! https://github.com/yceruto/symfony-ddd-skeleton/releases/tag/v5.4.1

The Symfony local server development compatible with this approach is now included in the README, thanks to @LucileDT!
